### PR TITLE
lab: Fix typos and punctuation

### PR DESCRIPTION
**Pull request by teodorjuravlea**
Signed-off-by: Teodor Juravlea <teodorjuravlea@gmail.com>

### CODE REVIEW

File: content/chapters/data/lab/content/investigate-memory.md [hunk 1]
```diff
@@ -379,7 +379,7 @@ This gives us an overview of when memory is allocated in Deluge / Python.
 
 [Servo](https://servo.org/) is a browser engine written in Rust that provides reusable components to implement web standards.
 
-We do not clone the repository since it's a very large one.
+We do not clone the repository, since it's a very large one.
```
Comment (teodutu) [inline content/chapters/data/lab/content/investigate-memory.md:382]:
```suggestion
We do not clone the repository, since it's very large.
```

**Review by teodutu — CHANGES_REQUESTED** [review]
Solid fixes once again. Thanks! I only have one small improvement to make.

A piece of advice: it would be good for you to make pull requests from branches other than `master` in the future. If you ever need to rebase them or work on something else, it's good to keep your `master` branch synced with the upstream repo.

And a question: do you use any form of spellchecker / grammar fixer (like maybe Grammarly) for this or do you come up with all changes manually? If it's the former, we may try to add it as a GitHub action and as a script for us to use ourselves.

**Comment by Alex-deVis** [comment]
> I'm using the browser extension from https://languagetool.org/ to find the fixes

I think a better approach would be to use a natural language linter like [textlint](https://textlint.github.io/). Seems like [super-linter](https://textlint.github.io/) supports it, but I have not investigated anything yet.
@teodutu, should we create an issue about this?

**Comment by teodutu** [comment]
> I think a better approach would be to use a natural language linter like [textlint](https://textlint.github.io/). Seems like [super-linter](https://textlint.github.io/) supports it, but I have not investigated anything yet.
> @teodutu, should we create an issue about this?

Go for it.

**Review by teodutu — APPROVED** [review]
:ok_hand:

**Pull request merged**